packages/@okta/vuepress-site/data/docs.yml

```diff
@@ -38,6 +38,8 @@
     - idps
     - linked-objects
     - policy
+    - risk-providers
```

Comment:
We do not want these pages showing up in navigation, so these should be removed.